showcase/echo_test.go

```diff
@@ -227,24 +244,26 @@ func TestPaginationWithToken(t *testing.T) {
 	str := "ab cd ef gh ij kl"
 	expected := strings.Split(str, " ")[1:]
 	req := &showcasepb.PagedExpandRequest{Content: str, PageSize: 2, PageToken: "1"}
```

Comment:
In https://github.com/googleapis/gapic-showcase/pull/806 I also added a `PagedExapndLegacy()` RPC that uses amx_results, so ideally we'd test that as well. (But that version of Showcase is not released yet, though we can do that).  I would add a TODO for now at least, and when we release Showcase (which may be after your mapped response PR and subsequent generator release!) we can come back and add the tests for this and for the mapped response, and re-release the generator. Chicken-and-egg problem!